framework/src/outputs/png/PNGOutput.C

```diff
@@ -134,8 +192,9 @@ PNGOutput::setRGB(png_byte * rgb, Real selection)
     case 2:
     // RWB.  Change spectrum.
     case 3:
-      color_spectrum_max = 1013; // 511;
+      color_spectrum_max = 511;
```

Comment:
Do you have any comments for these hardcoded values? They are odd.

Reply:
They are kinda odd.  I'll add explanatory comments.